ObjectiveGit/GTRepository.h

```diff
@@ -565,6 +565,20 @@ NS_ASSUME_NONNULL_BEGIN
 /// Returns whether `ahead` and `behind` were successfully calculated.
 - (BOOL)calculateAhead:(size_t *)ahead behind:(size_t *)behind ofOID:(GTOID *)headOID relativeToOID:(GTOID *)baseOID error:(NSError **)error;
 
+/// Creates an enumerator for walking the local commits in relative to the
+/// tracking branch.
+///
+/// Note that this will only be accurate iff `trackingBranch` truly is
+/// `localBranch`s tracking branch. If you need more general functionality, see
+/// -[GTBranch uniqueCommitsRelativeToBranch:error:].
```

Comment:
I think is only true because of the name you used. AFAICT, it's the same thing as `-[GTRepository enumerateUniqueCommitsUpToOID:relativeToOID:error:]` except that it doesn't do a time sort.